src/dparse/parser.d

```diff
@@ -3304,8 +3304,13 @@ class Parser
             advance();
             if (!currentIs(tok!"]"))
             {
+                node.indexer = parseAssignExpression();
+                if (node.indexer is null)
+                {
+                    goToBookmark(b);
+                    return node;
```

Comment:
This is never hit: 

![image](https://user-images.githubusercontent.com/4370550/35676826-99b66264-074d-11e8-8f1e-34d9f5314b6f.png)

Reply:
this is something else i have to put in fail files. Previously "uncoverage" was hidden by the mixin replaced by this fix.

Reply:
Actually there are plenty of granches made for errors that are not covered but working with files is boring... i gave a short eelier: see #185